Apps/Playground/package.json

```diff
@@ -18,7 +18,7 @@
     "@react-native-community/slider": "4.0.0-rc.2",
     "logkitty": "^0.7.1",
     "react": "^17.0.1",
-    "react-native": "^0.64.0-rc.0",
+    "react-native": "0.64.0-rc.0",
```

Comment:
Why is this change needed?

Reply:
when running react-native-windows initialization, react-native-windows will prompt/force users to use newer preview releases that support whatever react-native release is in the project. I don't think we are guaranteed that react-native-windows preview releases won't contain breaking changes (I have seen breaking changes already).

What I was seeing was that react-native would get installed at 0.64.0-rc.2, forcing us to update to react-native-windows 0.64.0-preview.7 which contained breaking changes for native modules (This was all during initialization prompts compared to an action of npm package installs). I believe we can ensure things will compile if we don't auto update to newer versions of react-native/react-native-windows. I think this will probably get better when react-native-windows has a formal 0.64.0 release.